srcs/layouts/guj_phonetic_in.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="ગુજરાતી ફોનેટિક - Gujarati Phonetic" script="gujarati">
+  <row>
+    <key key0="ટ" key1="`" key2="1" key3="!" key4="esc"/>
```

Comment:
The western numerals are on the keyboard instead of the gujarati ones. Is this intended ?